src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  static removeFromAlive(animal) {
+    Animal.alive = Animal.alive.filter((el) => el.health > 0);
+  }
```

Comment:
Almost done.
But why do we need the animal parameter in this method if it is not used?
Use more descriptive names in callbacks

```suggestion
  static removeFromAlive() {
    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
  }
```